src/itemBigData/ItemBigData.story.mdx

```diff
@@ -0,0 +1,50 @@
+import { Meta, Story, Preview, Props } from '@storybook/addon-docs/blocks'
+import { ItemBigData } from './index'
+
+<Meta title="Items|ItemBigData" />
+
+# ItemBigData
+
+<Preview>
+  <Story name="Default">
+    <ItemBigData mainTitle={'23452,2500000000'} />
+  </Story>
+</Preview>
+
+### With Additional Info
+
+<Preview>
+  <Story name="withAdditionalInfo">
+    <ItemBigData
+      mainTitle="23452,2500000000"
+      mainInfo="Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nam mollis elit ac tortor sollicitudin, nec sodales odio vestibulum. Proin dapibus nibh tortor, bibendum condimentum nunc aliquam et. Integer non turpis ac lacus congue ultrices vitae ut magna. Phasellus a quam a ante luctus cursus. Donec sit amet velit id ex consequat volutpat. Pellentesque habitant morbi tristique senectus et netus et malesuada fames ac turpis egestas. Fusce faucibus in ipsum eu blandit. Maecenas diam turpis, dictum nec turpis eget, lacinia hendrerit mauris. Sed vitae aliquet tellus. Pellentesque eget lacus sed leo tristique laoreet sed nec lorem. Quisque posuere tristique arcu ut feugiat. Phasellus dignissim dui justo, eu consequat lectus blandit at."
+      ariaLabel="Additional"
+    />
+  </Story>
+</Preview>
+
+---
+
+## Specifications
+
+### Normal state
+
+- Used to display a specific and important data.
+- They’re not clickable.
+
+### Long texts
+
+- **Data**: _it can’t drop line_.
+- **Additionnal Info**: there’s no limit on the number of lines.
+
+## Usage
```

Comment:
I'm wondering if we should just skip the Usage sections. The only added value is to show how to import (since we already have JSX examples for free from the stories) ?

Reply:
And Props! :)

Reply:
I'll take note and we are going to propose a template for documentation @wakooka